assets/css/screens/html-view.scss

```diff
@@ -0,0 +1,7 @@
+.card-report {
+  height: 100%;
+}
+
+.html-view {
+  height: inherit;
+}
```

Comment:
For screen, we can create a body class 👍 

Add the helper to layout view
```elixir
# layout_view.ex
def body_class(conn) do
  "#{module_class_name(conn)} #{action_name(conn)}"
end

defp module_class_name(conn) do
  controller_module(conn)
  |> Phoenix.Naming.resource_name("Controller")
  |> String.replace("_", "-")
end
```

Use in the layout template
```html
<body class="<%= body_class(@conn) %>">
```